account/identity.go

```diff
@@ -0,0 +1,128 @@
+package account
+
+import (
+	"time"
+
+	"github.com/goadesign/goa"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+	"golang.org/x/net/context"
+)
+
+// Identity ddenDescribes a unique Person with the ALM
+type Identity struct {
+	ID        uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field
+	CreatedAt time.Time
```

Comment:
@aslakknutsen Please embed `Lifecycle` struct and remove `CreatedAt`, `DeletedAt`, and `UpdatedAt`.